src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskController.java

```diff
@@ -289,4 +290,49 @@ public TaskController(CamundaService camundaService,
             .cacheControl(CacheControl.noCache())
             .build();
     }
+
+    @ApiOperation("Retrieve a list of Task resources identified by set of search"
+                  + " criteria that are eligible for automatic completion")
+    @ApiResponses({
+        @ApiResponse(
+            code = 200,
+            message = "OK",
+            response = GetTasksResponse.class
+        ),
+        @ApiResponse(
+            code = 400,
+            message = "Bad Request"
+        ),
+        @ApiResponse(
+            code = 403,
+            message = "Forbidden"
+        ),
+        @ApiResponse(
+            code = 415,
+            message = "Unsupported Media Type"
+        ),
+        @ApiResponse(
+            code = 500,
+            message = "Internal Server Error"
+        )
+    })
+    @PostMapping(path = "/auto-complete", produces = APPLICATION_JSON_VALUE)
```

Comment:
We should discuss this endpoint.  The naming is quite important as this is what ExUI will use and become part of the specification.